code/FulpstationCode/wintercoats.dm

```diff
@@ -0,0 +1,106 @@
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/science/head
+
+/obj/item/clothing/head/hooded/winterhood/science/head
+	mob_overlay_icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	icon = 'icons/fulpicons/phoenix_nest/wintercoathoods.dmi'
+	icon_state = "winterhood_rd"
+
+/obj/item/clothing/suit/hooded/wintercoat/engineering/head
+	name = "\improper Chief Engineer's winter coat"
+	desc = "this is the Chief Engineer's personal winter coat."
+	icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	mob_overlay_icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	icon_state = "wintercoat_ce"
+	item_state = "wintercoat_ce"
+	lefthand_file = 'icons/fulpicons/phoenix_nest/wintercoat_lefthand.dmi'
+	righthand_file = 'icons/fulpicons/phoenix_nest/wintercoat_righthand.dmi'
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 40, "fire" = 30, "acid" = 45) // 20 extra rad protection. Why not?
+	allowed = list(/obj/item/flashlight, /obj/item/assembly/flash/handheld, /obj/item/melee/classic_baton/telescopic, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/t_scanner, /obj/item/construction/rcd, /obj/item/pipe_dispenser, /obj/item/toy, /obj/item/storage/fancy/cigarettes, /obj/item/lighter)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/engineering/head
+
+/obj/item/clothing/head/hooded/winterhood/engineering/head
+	mob_overlay_icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	icon = 'icons/fulpicons/phoenix_nest/wintercoathoods.dmi'
+	icon_state = "winterhood_ce"
+
+
+/obj/item/clothing/suit/hooded/wintercoat/captain/hop
+	name = "\improper Head of Personel's winter coat"
+	desc = "this is the Head of Personel's personal winter coat. It has a small armor vest woven inside."
+	icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	mob_overlay_icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	icon_state = "wintercoat_hop"
+	item_state = "wintercoat_hop"
+	lefthand_file = 'icons/fulpicons/phoenix_nest/wintercoat_lefthand.dmi'
+	righthand_file = 'icons/fulpicons/phoenix_nest/wintercoat_righthand.dmi'
+	armor = list("melee" = 25, "bullet" = 25, "laser" = 25, "energy" = 10, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50) // Weaker armor vest. (-5% Melee, Bullet, Laser)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/captain/hop
+
+/obj/item/clothing/head/hooded/winterhood/captain/hop
+	mob_overlay_icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	icon = 'icons/fulpicons/phoenix_nest/wintercoathoods.dmi'
+	icon_state = "winterhood_hop"
+
+/obj/item/clothing/suit/hooded/wintercoat/cargo/head
+	name = "\improper Quarter Master's cargo winter coat"
+	desc = "It's just a regular cargo winter coat... it seems the Quarter Master still isn't a head of staff..."
+
+/* Commented out for now. This'll go into the QM's locker on Halloween or something.
+/obj/item/clothing/suit/cardborg/qm
```

Comment:
Just ignore this. This isn't the start of a backdoor to take over Fulp or anything.